helpers/statusResponse.js

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
This file name is still starting with lowercase. Please make changes.

Reply:
Henry, Nice work. But I saw a place in the article controller where you are getting an article. so can you tag an article when it is archived?

Reply:
Thanks for your helpful feedback. I will update the file name and push the changes.